### PR TITLE
Log generator: randomly spread logs between log levels

### PR DESCRIPTION
**Pull request by jordigh**
This should be a little more realistic instead of logging everything at the info log levels.

Also adds some basic linting.

### CODE REVIEW

File: log-generator/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+module.exports = {
+  extends: '@newrelic',
+  overrides: [
```
Comment (bizob2828) [inline log-generator/.eslintrc.js:9]: I wouldn't turn off this rule.  We try to require having the copyright header on every file

**Review by bizob2828 — APPROVED** [review]
nice additions to the script.

File: log-generator/.eslintrc.js (hunk 1 above)
Comment (michaelgoin) [reply log-generator/.eslintrc.js:9]: Agreed. If that file is failing... you probably just need to remove the newline between the `#!` and the `/*`

Comment (jordigh) [reply log-generator/.eslintrc.js:9]: Oh, look at that. I was following [this example](https://github.com/newrelic/node-newrelic/blob/main/.eslintrc.js#L18), but you're right, I can remove the blank line after the shebang instead.

**Pull request merged**